doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,305 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "aioConnectorMetadataSchemaVersion": "1.0.0",
+  "definitions": {
+    "additionalConfigurationType": {
+      "type": "object",
+      "description": "The schema for this 'additional configuration' field. Must provide inline schema xor the file that contains the schema",
+      "properties": {
+        "schema": {
+          "$ref": "http://json-schema.org/draft-07/schema#"
+        },
+        "schemaFile": {
+          "type": "string"
+        }
+      },
+      "oneOf": [
+        {
+          "required": [
+            "schema"
+          ]
+        },
+        {
+          "required": [
+            "schemaFile"
+          ]
+        }
+      ]
+    }
+  },
+  "properties": {
+    "aioConnectorMetadataSchemaVersion": {
+      "type": "string",
+      "description": "The version of AIO connector metadata schema that this connector metadata file adheres to."
+    },
+    "name": {
+      "type": "string",
+      "description": "The name of the connector."
+    },
+    "description": {
+      "type": "string",
+      "description": "A brief (human-readable) description of the connector."
+    },
```

Comment:
Could you pls also mention where this is shown? I would also mention that preference is to have it in english.